package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "contact-list",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.jsx",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build:dev": "webpack-dev-server --env dev",
+    "build:prod": "webpack-dev-server --env prod",
+    "lint": "eslint src/**/*.jsx"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/marishahv/ReactTask.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/marishahv/ReactTask/issues"
+  },
+  "homepage": "https://github.com/marishahv/ReactTask#readme",
+  "dependencies": {
+    "bootstrap": "^4.3.1",
+    "npm": "^6.9.0",
```

Comment:
Why do you need npm?

Reply:
Has been added accidentally. And should be removed.